frontend/tsconfig.json

```diff
@@ -1,21 +1,36 @@
 {
   "compilerOptions": {
     "allowJs": true,
+    "baseUrl": ".",
     "esModuleInterop": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "lib": ["dom", "es2020"],
-    "module": "commonjs",
+    "lib": [
+      "dom",
+      "es2020"
+    ],
+    "module": "esnext",
+    "moduleResolution": "node",
```

Comment:
this is needed since we use `"module": "esnext"` above now, so we don't get runtime error when running `jest`, which complains about not understanding `import/export` syntax